README.md

```diff
@@ -105,13 +105,15 @@ Follow instructions [here](https://github.com/microsoft/vscode-apimanagement/iss
 
 ## Managing Azure Subscriptions
 
-If you are not signed in to Azure, you will see a "Sign in to Azure..." link. Alternatively, you can select "View->Command Palette" in the VS Code menu, and search for "Azure: Sign In".
+If you are not signed in to Azure, you will see a "Sign in to Azure..." link.

```

Comment:
If you are not signed in to Azure, a 'Sign in to Azure...' link will appear.

Reply:
updated. thanks